weaver/store/mongodb.py

```diff
@@ -405,6 +407,7 @@ def save_job(self,
                  access=None,               # type: Optional[str]
                  notification_email=None,   # type: Optional[str]
                  accept_language=None,      # type: Optional[str]
+                 created_date=None,         # type: Optional[str]
```

Comment:
Can be named only `created` to match the property name, and receive directly a datetime as per my other comment.